mycli/main.py

```diff
@@ -419,8 +419,9 @@ def handle_editor_command(self, cli, document):
         saved_callables = cli.application.pre_run_callables
         while special.editor_command(document.text):
             filename = special.get_filename(document.text)
-            sql, message = special.open_external_editor(filename,
-                                                          sql=document.text)
+            sql, message = special.open_external_editor(
+                filename, sql=document.text,
+                default_text=self.get_last_query() or '')
```

Comment:
can `or ''` be removed here? I think `None` should be handled correctly in `special.open_external_editor`.

actually I'm not sure about `open_external_editor`, why does it need to decide to open `sql` or `default_text`?